examples/nextjs-lit/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Can this be newer in a Next app? We really don't want to compile out classes. And I don't see the custom-elements-es5-shim added anywhere... so is this setting even being used?